azure-pipelines.yml

```diff
@@ -257,9 +257,9 @@ stages:
           PLATFORM: 'x86-64'
           TEST_MODE: 'conda'
           PYTHON_VERSION: '3.9'
-        3.7 pip:
+        3.9 pip:
           TEST_MODE: 'pip'
-          PYTHON_VERSION: '3.7'
+          PYTHON_VERSION: '3.9'
```

Comment:
Can we maybe switch to 3.10 directly?

Reply:
Not easily until VTK makes a wheel. They should release 9.2 soon, which will have one